tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/TestMethodRunnerExecuteIntegration.cs

```diff
@@ -35,16 +35,11 @@ public class TestMethodRunnerExecuteIntegration
         /// <returns>Calltarget state value</returns>
         public static CallTargetState OnMethodBegin<TTarget>(TTarget instance)
         {
-            if (!MsTestIntegration.IsEnabled)
-            {
-                return CallTargetState.GetDefault();
-            }
-
-            return new CallTargetState((Scope)null, instance);
+            return CallTargetState.GetDefault();
```

Comment:
Shall we just omit this method if we don't need it?

Reply:
You're right, removing it.

Reply:
done in: 27f63fb